docs/metrics/geometry-matcher.md

```diff
@@ -0,0 +1,275 @@
+---
+search:
+  exclude: true
+---
+
+# Geometry Matcher
+
+Geometry matcher finds the best possible match, given the sets of the ground truth and prediction polygons for each
```

Comment:
We should use the term `inference` instead of `prediction` consistently. Is there any reason to prefer `prediction`?

Reply:
Good point. No specific reason.

Reply:
I've updated this and other guides to use `inference` instead of `prediction`

Reply:
Thanks. I also created a ticket to review all docs to update `prediction` to `inference`.  I think some images with the word `prediction` need to be updated as well.